.github/workflows/register_data_assets.yml

```diff
@@ -1,7 +1,12 @@
 name: Register Datasets
 
+# TODO restore
+# on:
+#   workflow_dispatch:
```

Comment:
is the intent to execute this on PR or to be triggered automatically? is the TOOD / commented out code needed?

Reply:
This workflow is ran manually. I need it to register the new data assets for the docker_taxi pipeline before the other new workflows can work. I will restore and remove this TODO once the workflows are past the storage account authentication issue. Let's leave this comment up to help me remember to do that :)

Reply:
All restored!